alacritty_terminal/src/term/mod.rs

```diff
@@ -1891,8 +1892,11 @@ impl<T: EventListener> Handler for Term<T> {
                     self.grid.clear_viewport(template);
                 }
 
-                self.selection =
-                    self.selection.take().filter(|s| !s.intersects_range(0..=last_line));
+                let last_visible_line_index = num_lines - 1;
```

Comment:
The variable name and definition is longer than the variable itself. What's the point?